tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -16,8 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 import unittest
+from typing import Any, Dict, List, Optional
 from unittest import mock
 
+from typing_extensions import TypedDict
```

Comment:
```suggestion
from airflow.typing_compat import TypedDict
```

Reply:
changed, thanks

Reply:
Actually @uranusjr , looks like changing this to airflow.typing_compat seems to change the behavior, it introduced these errors

`tests/providers/amazon/aws/operators/test_eks.py:88: error: Incompatible types in assignment (expression has type "Dict[str, Collection[Any]]", variable
has type "ClusterParams")
            self.create_cluster_params: ClusterParams = dict(
                                                        ^
tests/providers/amazon/aws/operators/test_eks.py:104: error: Incompatible types in assignment (expression has type "Dict[str, str]", variable has type
"NodeGroupParams")
            self.base_nodegroup_params: NodeGroupParams = dict(
                                                          ^
tests/providers/amazon/aws/operators/test_eks.py:125: error: Incompatible types in assignment (expression has type "Dict[str, Sequence[Any]]", variable has
type "BaseFargateProfileParams")
            self.base_fargate_profile_params: BaseFargateProfileParams = dict(
                                                                         ^
tests/providers/amazon/aws/operators/test_eks.py:183: error: Incompatible types in assignment (expression has type "Dict[str, Sequence[Any]]", variable has
type "CreateFargateProfileParams")
            self.create_fargate_profile_params: CreateFargateProfileParams = dict(
                                                                             ^
tests/providers/amazon/aws/operators/test_eks.py:205: error: Incompatible types in assignment (expression has type "Dict[str, Sequence[str]]", variable has
type "CreateNodegroupParams")
`